index.php

```diff
@@ -0,0 +1,33 @@
+<?php
+    include "includes/header.php";
+
+$weight = 173;
+echo "<br>";
+$age= '32';
+echo "<br>";
+$first = 'Mahougbe';
+echo "<br>";
+$last = 'HOUFFON';
+echo "<br>";
+$kg = 'Num';
+Function poundsToKG ($kg){
```

Comment:
the idea of the function was to bring in the weight in pounds as an argument. I try to keep my arg names unique from my vars - so let's call that $w. I would have written something like:

Function poundsToKG ($w) {
    $kg = $w * (1/2.2);  //$kg is scoped here to the function. I try to avoid global calls unless necessary.
    return $kg;  //this will make that value availalbe to the larger program - but I'll have to store it
}
$result = poundsToKG($weight); //see how I pass in the lb value? and I'll get back the KG value.

I hope this makes sense for you. I won't take off since what you have works - but it isn't really an ideal solution. Let me know if you have questions 😄